Makefile.cache

```diff
@@ -283,7 +283,7 @@ define SAVE_INTO_CACHE
 	$(call GET_MOD_SHA,$(1))
 
 	# Form the cache file name
-	$(eval $(1)_MOD_CACHE_FILE  := $(1)-$($(1)_DEP_MOD_SHA)-$($(1)_MOD_HASH).tgz)
+	$(eval $(1)_MOD_CACHE_FILE  := $(1)-$($(1)_DEP_MOD_SHA)-$($(1)_MOD_HASH))
```

Comment:
Why rename the cache file? The cache file loaded should be the same as it saved, right?

Reply:
had to do it because the compression step was done twice. Changed the approach to use a .cached.log. This is also handled